packages/tocco-ui/src/SearchBox/SearchBox.js

```diff
@@ -42,24 +44,28 @@ class SearchBox extends React.Component {
 
   render() {
     return (
-      <form onSubmit={this.handleSubmit} className="tocco-searchbox">
-        <div className="input-group">
-          <input
-            type="text"
-            className="form-control"
-            placeholder={this.props.placeholder}
-            value={this.state.inputValue}
-            onChange={this.onChange}
-          />
-          <StyledInputGroupBtn>
-            <Button
-              icon="search"
-              iconPosition="sole"
-              type="submit"
+      <StyledSearchBoxForm onSubmit={this.handleSubmit}>
+        <StatedValue
+          hasValue={!!this.state.inputValue && !!this.state.inputValue.length > 0}
```

Comment:
` !!this.state.inputValue.length > 0 ` is not needed? Or what values do you handle with that